apps/fz_http/test/fz_http/repo/notifier_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule FzHttp.Repo.NotifierTest do
+  use ExUnit.Case, async: false
+
+  import FzHttp.TestHelpers
+
+  alias Ecto.Adapters.SQL.Sandbox
+  alias FzHttp.Events
+  alias FzHttp.Repo
+
+  setup do
+    start_supervised!({Postgrex.Notifications, [name: Repo.Notifications] ++ Repo.config()})
+    start_supervised!(Repo.Notifier)
+    :ok = Sandbox.checkout(Repo)
+  end
+
+  test "adds and removes user from wall state on db notifications" do
+    Sandbox.unboxed_run(Repo, fn ->
+      {:ok, [user: user]} = create_user(%{})
+
+      Process.sleep(250)
+
+      wall_state_add = :sys.get_state(Events.wall_pid())
+
+      Repo.delete!(user)
+
+      Process.sleep(250)
+
+      wall_state_remove = :sys.get_state(Events.wall_pid())
+
+      assert wall_state_add ==
+               %{
+                 users: MapSet.new([user.id]),
+                 devices: MapSet.new([]),
+                 rules: MapSet.new([])
+               }
+
+      assert wall_state_remove ==
+               %{
+                 users: MapSet.new([]),
+                 devices: MapSet.new([]),
+                 rules: MapSet.new([])
+               }
+    end)
+  end
+
+  test "adds and removes rule from wall state on db notifications" do
+    Sandbox.unboxed_run(Repo, fn ->
+      {:ok, [rule: rule]} = create_rule(%{})
+
+      Process.sleep(250)
```

Comment:
250ms seems like a lot?

Reply:
Yeah, I wasn't exactly sure how long it takes for a notify to come back from pg, but I can shorten this and see.

Reply:
With no network latency it should be very fast, probably under 5ms. But maybe 10-50ms is a good starting point here since this is GH Actions after all